src/layers/medWidgets/process/single_filter/medAbstractBiasCorrectionProcessPresenter.h

```diff
@@ -11,19 +12,14 @@
 
 =========================================================================*/
 
-#pragma once
-
+#include <medAbstractBiasCorrectionProcess.h>
 #include <medAbstractSingleFilterOperationProcessPresenter.h>
 #include <medProcessPresenterFactory.h>
-#include <medAbstractBiasCorrectionProcess.h>
-
 #include <medWidgetsExport.h>
 
 class MEDWIDGETS_EXPORT medAbstractBiasCorrectionProcessPresenter: public medAbstractSingleFilterOperationProcessPresenter
 {
     Q_OBJECT
-private:
```

Comment:
It's not a problem to keep empty "privet, protected, public" if it's done to have pre-establish sections